cmd/expiration-mailer/main.go

```diff
@@ -365,7 +377,10 @@ func (m *mailer) findExpiringCertificates(ctx context.Context) error {
 		}
 
 		processingStarted := m.clk.Now()
-		m.processCerts(ctx, certs)
+		err = m.processCerts(ctx, certs)
+		if err != nil {
+			m.log.AuditErrf("%s", err)
```

Comment:
Probably fine either way:
```suggestion
			m.log.AuditErr(err.Error())
```